frontends/mit-learn/src/pages/ChannelPage/TopicChannelTemplate.tsx

```diff
@@ -0,0 +1,285 @@
+import React from "react"
+import {
+  styled,
+  Breadcrumbs,
+  Banner,
+  ChipLink,
+  Typography,
+  Skeleton,
+} from "ol-components"
+import { SearchSubscriptionToggle } from "@/page-components/SearchSubscriptionToggle/SearchSubscriptionToggle"
+import { useChannelDetail } from "api/hooks/channels"
+import ChannelMenu from "@/components/ChannelMenu/ChannelMenu"
+import ChannelAvatar from "@/components/ChannelAvatar/ChannelAvatar"
+import { LearningResourceTopic, SourceTypeEnum } from "api"
+import { HOME as HOME_URL } from "../../common/urls"
+import {
+  CHANNEL_TYPE_BREADCRUMB_TARGETS,
+  ChannelControls,
+} from "./ChannelPageTemplate"
+import MetaTags from "@/page-components/MetaTags/MetaTags"
+import { ChannelTypeEnum, TopicChannel } from "api/v0"
+import {
+  useLearningResourceTopic,
+  useLearningResourceTopics,
+} from "api/hooks/learningResources"
+import { propsNotNil } from "ol-utilities"
+import invariant from "tiny-invariant"
+
+const ChildrenContainer = styled.div(({ theme }) => ({
+  paddingTop: "40px",
+  [theme.breakpoints.down("sm")]: {
+    paddingTop: "24px",
+  },
+}))
+
+const ChannelControlsContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "end",
+  flexGrow: 0,
+  flexShrink: 0,
+  order: 2,
```

Comment:
`order` is usually something we should avoid, since it's visual only and doesn't affect keyboard nav. And here I don't think it does anything:

<img width="1235" alt="Screenshot 2024-10-08 at 10 19 17 AM" src="https://github.com/user-attachments/assets/61c663fb-3cd1-43ce-aec2-fcb2089c4994">

Reply:
This got copied over from `DefaultChannelTemplate`: https://github.com/mitodl/mit-open/blob/d9ed99aa0f15f528f2ddbf9a0f963cdccfafd9a6/frontends/mit-learn/src/pages/ChannelPage/DefaultChannelTemplate.tsx#L28

My best guess is that this was done to try and control the position of the channel controls container. I agree that it doesn't do anything and have removed it from both places.